### DISCUSSION TITLE
vscode-go installing doesn't work

### DISCUSSION THREAD

**Discussion by NX-Luca-Frangiamore** [mfussenegger/nvim-dap#1209] [Q&A]
### Debug adapter definition and debug configuration

when i follow the istruction i get stuck in: npm i
since it doesn't see package.json

### Steps to Reproduce

git clone https://github.com/golang/vscode-go
cd vscode-go
npm install <- It is broken here

### Expected Result

install the dependency npm

### Actual Result

This is related to npm not being able to find a file.

**Comment by mfussenegger** [comment]
The wiki is community maintained - if the instructions no longer work people need to figure out what changed and update it. 

The issue tracker is for client bugs.

**Comment by jZhangTk** [comment]
I had the same issue and just found a fix.

I think the folder structure is changed. We now need to `cd vscode-go/extension` and then do the same.

I have also updated the wiki page to reflect the change.
